etna/ciim/constants.py

```diff
@@ -704,6 +704,9 @@ class Display(StrEnum):
 TNA_URLS = {
     "discovery_browse": "https://discovery.nationalarchives.gov.uk/browse/r/h",
     "tna_accessions": "https://www.nationalarchives.gov.uk/accessions",
+    "discovery_rec_default_fmt": "https://discovery.nationalarchives.gov.uk/details/r/{iaid}",
+    "discovery_rec_archon_fmt": "https://discovery.nationalarchives.gov.uk/details/a/{iaid}",
+    "discovery_rec_creators_fmt": "https://discovery.nationalarchives.gov.uk/details/c/{iaid}",
```

Comment:
Hi Allan, nice work, just one small question - happy for it to stay as is, just one little thing.

Why `fmt` and not `format`? I think it's fairly clear, but that's because I have an understanding of the codebase and what's going on. For someone reading this for the first time, it might not be overly clear what `fmt` means. Using `format` instead of `fmt` only adds another 3 letters, so why shorten it? 🙂 

Happy for this to be accepted as is, you can decide whether you want `fmt` or `format` 🙂

Reply:
On the other hand, the name would be too big to look if expanded - see rec is also shortened.

Reply:
Fair point @TNA-Allan 😃 Happy for it to be merged etc